### PR TITLE
Develop to Master

### PR DESCRIPTION
**Pull request by chiragjn**
- Name detection pattern fix
- Python 2/3 iterables compatibility changes

### CODE REVIEW

File: datastore/elastic_search/create.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-from utils import filter_kwargs
+from .utils import filter_kwargs
```
Comment (srajdev) [inline datastore/elastic_search/create.py]: W1618 [34mimport missing `from __future__ import absolute_import`[0m

File: lib/nlp/levenshtein_distance.py [hunk 2]
```diff
@@ -1,3 +1,5 @@
+from six.moves import range
```
Comment (srajdev) [inline lib/nlp/levenshtein_distance.py]: W1618 [34mimport missing `from __future__ import absolute_import`[0m

File: ner_v1/detectors/textual/name/name_detection.py [hunk 3]
```diff
@@ -521,7 +522,7 @@ def remove_emojis(self, text):
         Returns:
             text (str): text with emojis replaced with ''
         """
-        emoji_pattern = re.compile(ur'[{0}]+'.format(''.join(EMOJI_RANGES.values())), re.UNICODE)
+        emoji_pattern = re.compile(ur'[{0}]+'.format(''.join(list(EMOJI_RANGES.values()))), re.UNICODE)
```
Comment (srajdev) [inline ner_v1/detectors/textual/name/name_detection.py]: E1611 [1;31municode raw string literals not supported in 3.x[0m

**Comment by haptik-deployment** [comment]
UNIT TESTS HAVE PASSED... Good To Merge

**Pull request merged**